ui/src/StateContainer.js

```diff
@@ -122,6 +128,30 @@ export class StateContainer extends LitElement {
         this.cookieService.makeUserInfoCookie(e.detail.userInfo);
     }
 
+    handleUpdateToast(e) {
+        let toast;
+
+        if (e.detail.state !== ToastStates.INACTIVE) {
+            toast = {
+                state: e.detail.state,
+                message: e.detail.message,
+            };
+        } else {
+            toast = {
+                state: ToastStates.INACTIVE,
+            };
+        }
+
+        this.toast = toast;
```

Comment:
When will we have a toast with an inactive state and a message? If that never occurs, we can remove the toast.state and depend on the existence of toast.message to show or hide the toast.